languages.yaml

```diff
@@ -79,6 +79,22 @@ Ada:
       - "#0018C9"
       - "#0C0A7C"
     chip: "#02F88C"
+Arduino:
+  type: programming
+  ascii: |
+    {0}   :-=====-:       .-=====-:   
+    {0} :===-:::-===-.  -===-:::-===: 
+    {0}:==-       .-==-===:  .:   :==-
+    {0}===  .----   :===-   :==-.  ===
+    {0}-==.        :=====-   .:   .==-
+    {0} -==-.. .:-===: :===-:. ..-=== 
+    {0}  .-=======-.     .-=======-:      
+  colors:
+    ansi:
+      - cyan
+    hex:
+      - '#189BA1'
+    chip: "#F34B7D"
```

Comment:
Hi @Sh4rk-Byte 

I cleaned your design a bit, tell me what you think:

![image](https://github.com/o2sh/onefetch/assets/13710835/93a4702d-63ef-456a-80cb-4e7846423871)

the code:

```
Arduino:
  type: programming
  ascii: |
    {0}     ,=======.    ,=======.
    {0}    //       \\  //       \\
    {0}   //         \\//    #    \\
    {0}   ||   ###    //    ###   ||
    {0}   \\         //\\    #    //
    {0}    \\       //  \\       //
    {0}     `======='    `======='
    {0}      __   __       ___       __
    {0} /\  |__) |  \ |  |  |  |\ | /  \
    {0}/--\ |  \ |__/ \__/ _|_ | \| \__/
  colors:
    ansi:
      - cyan
    hex:
      - "#00838B"
    chip: "#F34B7D"
```

Reply:
@o2sh FYI I just expanded the ASCII art in https://github.com/o2sh/onefetch/pull/1176/commits/0aa16511eb1d30e59c6ce57887dd1d56222204d0. I think @Sh4rk-Byte shrunk it to handle the "line to large" compilation error, but it was triggered by trailing whitespace, not a problem with the actual art.

Reply:
@o2sh yeah I like your ASCII art, it's much clearer. Thanks